salt/vault/sync.sls

```diff
@@ -0,0 +1,23 @@
+{% from "vault/map.jinja" import vault with context %}
+
+Sync Vault Policies:
+  module.run:
+    - vault.policies_synced:
```

Comment:
hrm, ok, interesting approach, but... Rather than rely on `module.run`, _usually_, a salt module is an _action_, i.e. `set_policies`, `get_policies`, `diff_policies`. A salt state has the stateful logic around "what is the current state, what is the user wanting, let's apply only the diff"... so a name like `policies_synced` is more of a salt state...